### ISSUE TITLE
Are automated transforms a possibility?

### ISSUE THREAD

**Issue by DamnedScholar** [zsviczian/obsidian-excalidraw-plugin#337]
**Is your feature request related to a problem? Please describe.**
It would be nice to control the size and positioning of elements that already exist programmatically, and some of the clip art is just too big for the window by default (and is hard to resize).

**Describe the solution you'd like**
Excalidraw has [methods to transform elements](https://github.com/excalidraw/excalidraw/blob/master/src/element/resizeElements.ts#L52), but they don't seem to be exposed on the view or element interfaces. Since a fork of the package is maintained for this plugin, it might not be hard to patch them on to something that gets exported under `ExcalidrawAutomate`. The umbrella function `transformElements()` refers to the conditions of the pointer when passing arguments to individual transforms, but the individual transforms _don't_ care.

**Comment by zsviczian** [comment]
I am keeping custom changes to a minimum as every single change makes it harder and harder to deploy core product changes to Obsidian. That said, I think exposing this function on ExcalidrawAutomate is relatively low risk. I'll look into this a bit more, but will probably do it.

**Comment by DamnedScholar** [comment]
Yeah, how to shift a virtual shape across a virtual grid is a pretty solved problem, so the transform methods are unlikely to see dramatic (if any) changes.
